tests/testthat/test-get-country-regional-data.R

```diff
@@ -0,0 +1,53 @@
+# Tests each dataset returned by countries specified in get_info_covidregionaldata
+source("custom_tests/get_data.R")
+
+# 1. Test data.frame is returned
+test_that("Level 1 data returned in dataframes", {
+  skip_on_cran()
+  
+  level1_dataframes <- purrr::map(data_level_1, ~ class(.x)) %>%
```

Comment:
what happens if a datasource has caused an error in get_data here? I imagine accessing needs to happen safely (using try catch, purrr::safely or similar)?